sklearn/model_selection/_search.py

```diff
@@ -967,19 +967,19 @@ class GridSearchCV(BaseSearchCV):
                                         mask = [ True  True False False]...),
             'param_degree': masked_array(data = [2.0 3.0 -- --],
                                          mask = [False False  True  True]...),
-            'split0_test_score'  : [0.8, 0.7, 0.8, 0.9],
-            'split1_test_score'  : [0.82, 0.5, 0.7, 0.78],
-            'mean_test_score'    : [0.81, 0.60, 0.75, 0.82],
-            'std_test_score'     : [0.02, 0.01, 0.03, 0.03],
+            'split0_test_score'  : [0.80, 0.70, 0.80, 0.93],
```

Comment:
Now the last value is not consistent with the table above ... either update the table or use 0.9 and make sure that mean and std match.

Reply:
Thanks, I didn't double checked